plugins/nexus-repository-manager/src/types.ts

```diff
@@ -9,20 +9,41 @@ export type Annotation = {
   query?: (str: string) => SearchServiceQuery;
 };
 
-export type RawAsset = {
-  schemaVersion: number;
-  mediaType: string;
-  config: Config;
-  layers: Layer[];
+export type RawAsset = RawAssetSchema1 | RawAssetSchema2;
+
+// https://docs.docker.com/registry/spec/manifest-v2-1/
```

Comment:
Small nit: Can you please change this to:

```suggestion
/** @see {@link https://docs.docker.com/registry/spec/manifest-v2-1/|Image Manifest Version 2, Schema 1} */
```

This would allow people to get an IDE hint (in IDEs that support JSDoc) along with the type. 🙂

#### Before
![image](https://github.com/janus-idp/backstage-plugins/assets/7453394/cec8df7a-507c-47d5-ab15-7596f12ea543)



#### After
![image](https://github.com/janus-idp/backstage-plugins/assets/7453394/aff2ca18-f809-4486-81ce-c1b4c905249d)

Reply:
Oh, this is very nice! Will be using this in the future. Updated.